src/Microsoft.Extensions.Logging.Console/ConsoleLoggerProvider.cs

```diff
@@ -22,9 +22,10 @@ public class ConsoleLoggerProvider : ILoggerProvider, ISupportExternalScope
         private static readonly Func<string, LogLevel, bool> falseFilter = (cat, level) => false;
         private IDisposable _optionsReloadToken;
         private bool _includeScopes;
+        private bool _disableColors;
         private IExternalScopeProvider _scopeProvider;
 
-        public ConsoleLoggerProvider(Func<string, LogLevel, bool> filter, bool includeScopes)
+        public ConsoleLoggerProvider(Func<string, LogLevel, bool> filter, bool includeScopes, bool disableColors = false)
```

Comment:
This looks like a breaking change. I recommend adding a 2nd constructor that has the extra parameter, and then chain the two constructors.

Reply:
Addressed in 2bfbdfb2d5356bc7145a47dff05f953b99b76279